.evergreen/config/generate-config.php

```diff
@@ -0,0 +1,85 @@
+#!/bin/env php
+<?php
+
+// Supported PHP versions. Add new versions to the beginning of the list
+$supportedPhpVersions = ['8.2', '8.1', '8.0', '7.4'];
```

Comment:
What do you think about splitting this across lines to make diffs more readable? Likewise for `$supportedMongoDBVersions` below.